triple-store/triple-store-test/src/test/java/com/powsybl/triplestore/test/TripleStoreSubqueryLimit1Test.java

```diff
@@ -0,0 +1,60 @@
+package com.powsybl.triplestore.test;
+
+import com.powsybl.triplestore.api.QueryCatalog;
+import com.powsybl.triplestore.api.TripleStoreFactory;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class TripleStoreSubqueryLimit1Test {
```

Comment:
The authorship is missing:

```suggestion
/**
 * @author Luma Zamarreño {@literal <zamarrenolm at aia.es>}
 */
class TripleStoreSubqueryLimit1Test {
```